src/molecules/formfields/CreditCardField/__tests__/credit_card_field.spec.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+// import sinon from 'sinon';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+import CreditCardField from 'molecules/formfields/CreditCardField';
+
+describe('<CreditCardField />', () => {
```

Comment:
@drewdrewthis just the commented out code - although I agree we should have a better standard test because the plop generated file isn't really doing anything at the moment